web/src/api/index.ts

```diff
@@ -23,6 +53,24 @@ function delay(amount: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, amount));
 }
 
+export async function logIn(
+  username: string,
+  password: string,
+): Promise<string | undefined> {
+  const response = await client.post('/login', {
+    headers: { Authorization: 'Basic ' + btoa(username + ':' + password) },
```

Comment:
Use template strings, aka backticks:

```ts
headers: { Authorization: `Basic ${btoa(`${username}:${password}`}` }
```

Reply:
c605f78ae3faa4228e0a69e2dd09b21a8610246d

Had some issues before, but they turned out to be unrelated to template strings in the end. Only using a single level though, Emacs seems to get confused by nested templates (even if they're technically legal).

Reply:
Mhm bummer. Let's keep it like this and eventually update it.